ansible/operation-apply-license.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Apply Redpanda License
+  hosts: redpanda
+  become: true
+  vars:
+    rpk_bin: rpk
+
+  tasks:
+    - name: Check cluster health
+      ansible.builtin.shell: |
+        {{ rpk_bin }} cluster health | grep -i 'healthy:' | tr -d '[:space:]' | awk -F ':' '{print tolower($2)}'
```

Comment:
Probably better to do something like this and check the return code of grep, rather than trying to parse out the true/false status.

rpk cluster health | grep -q -i 'healthy.*false'

returns 0 if one or more lines matched the regex. returns 1 if zero lines matched.

Reply:
(though, it would be better if rpk cluster health set a return code of 0 or 1 depending on cluster health state, then we could check the return code directly -- don't think that support exists right now)